src/components/ActivityLog/MapSplitView.tsx

```diff
@@ -101,6 +101,28 @@ export default function MapSplitView({
     });
   }, [activities, activityMarkerHighlighted, onActivityMarkerClickCallback]);
 
+  const { plantingSites } = useProjectPlantingSites(projectId);
+
```

Comment:
NIT it might just be a personal preference, but it feels weird to me to have useCallbacks defined above variables that are being used in the component (such as `plantingSites`)